src/components/SidePanel/SidePanel.test.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import user from '@testing-library/user-event'
+import SidePanel from './'
+
+jest.useFakeTimers()
+
+describe('Show / no show', () => {
+  test('should be shown or hidden using show prop using useAnimatedRender under the hood', () => {
+    const { queryByTestId, rerender } = render(<SidePanel />)
+
+    expect(queryByTestId('sidepanel-overlay')).not.toBeInTheDocument()
+
+    rerender(<SidePanel show />)
+    fireEvent.animationEnd(queryByTestId('sidepanel-overlay'))
+
+    expect(queryByTestId('sidepanel-overlay')).toBeInTheDocument()
+    expect(
+      queryByTestId('sidepanel-overlay').classList.contains('element-in')
+    ).toBe(true)
```

Comment:
What do you think about using `toHaveClass` assertion function? It would give better error message if failed :)

Reply:
I wonder if we have that in HSApp, it's a good idea, though I'd like to add those in a different PR and perhaps add them to HSApp (or bring them from there)

Reply:
I have been using them, so they are for sure :smile: It's from `@testing-library/jest-dom` library that is set up in `config/jest/setupTests.js` in hsds repo already

Reply:
Talk about not knowing what we have... thanks! (I changed the assertions)